tests/test_pipeline_run_dvc.py

```diff
@@ -0,0 +1,114 @@
+import subprocess
+from pathlib import Path
+
+import pytest
+
+from lineapy.api.models.linea_artifact import get_lineaartifactdef
+from lineapy.data.types import PipelineType
+from lineapy.graph_reader.artifact_collection import ArtifactCollection
+from lineapy.plugins.pipeline_writer_factory import PipelineWriterFactory
+
+
+@pytest.mark.skipif(
+    subprocess.check_call(["git", "--version"]) != 0,
+    reason="dvc requires git to be installed",
+)
+@pytest.mark.dvc
+@pytest.mark.slow
+@pytest.mark.parametrize(
+    "input_script1, input_script2, artifact_list, pipeline_name, dependencies, dag_config, input_parameters",
+    [
+        pytest.param(
+            "housing",
+            "",
+            ["y", "p value"],
```

Comment:
It may be possible there are no dependencies between artifacts, but no configurations test them. Is it not necessary to test that behavior here?

Reply:
I think given how expensive each of these pipeline run tests are (over a minute locally on my machine) we want to try to test as many features as possible and be strategic about how we configure the few tests we can run. 

I think for each of these, 3-4 tests is the maximum, and I would rather test the different behaviors of having dependencies which require inter task communication.